lib/helpers/getDomNodeProfile.js

```diff
@@ -13,23 +15,31 @@ const getTextContent = (el, options) => {
 }
 
 export function getDomNodeProfile(el, options = {}) {
-  return {
-    action: el.action,
+  let profile = {
     class: el.className,
     href: el.href || null,
     id: el.id,
-    method: el.method,
     name: el.name,
-    ng_click: el.getAttribute('ng-click'),
-    ng_model: el.getAttribute('ng-model'),
+    all_attrs:  getDomNodeAttributes(el, /^((?!value).)*$/),
     node_name: el.nodeName,
-    selector: getDomNodePath(el),
     tag_name: el.tagName,
     text: el.text,
-    text_content: getTextContent(el, options),
     title: el.title,
-    type: el.type,
-    x_position: el.offsetLeft || el.clientLeft || null,
-    y_position: el.offsetTop || el.clientTop || null
+    type: el.type
   };
+
+  if (!options.basicProfile) {
```

Comment:
What's a basic profile mean here? It looks like it's a regular profile but with extra information

Reply:
basic profile was meant to be the opposite. Basic is just a small list of properties that wilson wanted on the n-parents, rather than everything. The boolean logic does make it confusing, and hopefully moving it out into it's own function clears it up a bit